### PR TITLE
Translate "Support of Ruby 2.3 has ended" (ja)

### PR DESCRIPTION
**Pull request by acomagu**

### CODE REVIEW

File: ja/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: "Ruby 2.3 のサポートが終了しました"
+author: "antonpaisov"
+translator: "acomagu"
+date: 2019-03-31 00:00:00 +0000
+lang: ja
+---
+
+Ruby 2.3 系列のすべてのサポートが終了しました。
+
+2018 年の 3 月 28 日に Ruby 2.3.7 がリリースされた後、 Ruby 2.3 系列のサポートはセキュリティメンテナンスの段階に入っていました。それから 1 年が経ち、それも終わろうとしています。 2019 年 3 月 31 をもって Ruby 2.3 系列の全てのサポートは終了し、 2.4 以降の脆弱性やバグの修正が 2.3 にバックポートされることもなくなります。 2.3 用の一切のパッチも作成されません。 Ruby 2.6 や 2.5 へのアップグレードを強く推奨します。
```
Comment (toru-takahashi) [inline ja/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md]: just a typo; 2019 年 3 月 31 日

Comment (acomagu) [reply ja/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md]: Thank you!!!!!! :bow: :bow:
76000dd

**Comment by takahashim** [comment]
Thanks for your PR!

> Ruby 2.6 や 2.5 へのアップグレードを強く推奨します。

Could you add a phrase`as soon as possible` in the original (e.g. `速やかに` ?)

**Comment by acomagu** [comment]
@takahashim Thank you for reviewing!! :pray:
Fixed: a055a51

**Comment by JuanitoFatas** [comment]
皆様あいがとうございました! Thanks all.

**Pull request merged**